src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+	String getPrintableString();
```

Comment:
do you think you can take advantage of getPrintableString to iterate through a list of Printable objects?